app/src/main/java/com/lagradost/cloudstream3/ui/home/HomeFragment.kt

```diff
@@ -521,15 +521,13 @@ class HomeFragment : Fragment() {
             }
         }
 
-        //Disable Random button, if its toggled off on settings
+        //Load value for toggling Random button. Hide at startup
         context?.let {
             val settingsManager = PreferenceManager.getDefaultSharedPreferences(it)
             toggleRandomButton =
                 settingsManager.getBoolean(getString(R.string.random_button_key), false)
-            home_random?.isVisible = toggleRandomButton
-            if (!toggleRandomButton) {
-                home_random?.visibility = View.GONE
-            }
+            home_random?.isVisible = false
+            home_random?.visibility = View.GONE
```

Comment:
why duplicate call here?

Reply:
No reason. Just copied the line out of the ``if`` condition. Will remove the `home_random?.isVisible = false` though